src/components/victory-area/victory-area.js

```diff
@@ -23,6 +23,42 @@ export default class VictoryArea extends React.Component {
   static role = "area";
 
   static defaultTransitions = {
+    onLoad: {
```

Comment:
onLoad, onExit, and onEnter are the same for victory-line and victory-area. Can some of this logic get extracted into victory-core?